test/sharness/t0081-repo-pinning.sh

```diff
@@ -238,9 +238,9 @@ test_expect_success "some are no longer there" '
 '
 
 test_expect_success "recursive pin fails without objects" '
-	ipfs pin rm "$HASH_DIR1" &&
-	test_must_fail ipfs pin add -r "$HASH_DIR1" --timeout=500ms 2>err_expected8 &&
-	grep "context deadline exceeded" err_expected8 ||
+	ipfs pin rm -r=false "$HASH_DIR1" &&
```

Comment:
The name of the test implies it tests recursive pinning.

Maybe more specifically: why has this test been changed?